book/typst/modules/module5-exercises.typ

```diff
@@ -15,15 +18,185 @@
   )
 
 
+
+
+  book_only(pagebreak())
   question(
     slide[
+      #align(center, block[
+        #image("../images/slope-field-3d.png",width: 2in)
+
+        #link("https://www.desmos.com/3d/kvyztvmp0g")[https://www.desmos.com/3d/kvyztvmp0g]
```

Comment:
No need for the `[...]` part. If you leave it blank it is populated with the URL automatically. Only use it if you want to override the text for the link.